CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 2.0
+
+## 2.0.0 - 2024-04-26
+
+### Added
+
+- Support `LoadBalancerIPMode` in AntreaProxy to implement K8s [KEP-1860](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1860-kube-proxy-IP-node-binding). ([#6102](https://github.com/antrea-io/antrea/pull/6102), [@hongliangl])
+- Add `sameLabels` field support for Antrea ClusterNetworkPolicy peer Namespace selection to allow users to create ACNPs that isolate Namespaces based on their label values. ([#4537](https://github.com/antrea-io/antrea/pull/4537), [@Dyanngg])
+- Add multiple physical interfaces support for secondary network bridge. ([#5959](https://github.com/antrea-io/antrea/pull/5959), [@aroradaman])
```

Comment:
```suggestion
- Add multiple physical interfaces support for a secondary network bridge. ([#5959](https://github.com/antrea-io/antrea/pull/5959), [@aroradaman])
```

Reply:
Change to `interfaces support for the secondary network bridge.`